schema/partners.js

```diff
@@ -36,8 +37,7 @@ const Partners = {
       `,
     },
     type: {
-      type: new GraphQLList(GraphQLString),
-      description: 'Only return partners of the specified _type(s).',
+      type: new GraphQLList(PartnerClassification.type),
```

Comment:
yeah agree, it's just that `type` is the param that the api is expecting and returning. we'd have to do some explicit mapping if we wanted to do what you're saying.